website/blog/2024-03-11-AutoDefense/Defending LLMs Against Jailbreak Attacks with AutoDefense.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: AutoDefense - Defend against jailbreak attacks with AutoGen
+authors: 
+- yifanzeng
+- yiranwu
+tags: [LLM, GPT, research]
+---
+
+![image-20240306191435555](imgs/image-20240306191435555.png)
+
+## TL;DR
+
+- We propose **AutoDefense**, a multi-agent defense framework using AutoGen to protect LLMs from jailbreak attacks.
+- AutoDefense employs a response-filtering mechanism with specialized LLM agents collaborating to analyze potentially harmful responses.
+- Experiments show our three-agent defense system with LLaMA-2-13B effectively reduces jailbreak attack success rate while maintaining low false positives on normal user requests.
+
+
+## The AutoDefense Framework 
+
+AutoDefense is a multi-agent defense framework built on AutoGen that filters harmful responses from LLMs. It consists of three main components:
```

Comment:
```suggestion
Despite extensive pre-training in moral alignment to prevent generating harmful information at user request, large language models (LLMs) remain vulnerable to jailbreak attacks. 
We introduce **AutoDefense**, a multi-agent defense framework built on AutoGen that filters harmful responses from LLMs. 
Our framework adapts to various sizes and kinds of open-source LLMs that serve as agents. 

AutoDefense consists of three main components:
```